packages/prop-controllers/README.md

```diff
@@ -0,0 +1 @@
+This package contains deprecated prop-controllers and is intended for internal use only.
```

Comment:
Let's add a title to this README and maybe some more information as to why it's needed?